### PR TITLE
 Issue with floating buttons

### LINKED ISSUE #2800: FloatingButton in FloatingButtonGroup should have adaptable borders
## Bug Description

image

The borders of a FloatingButton inside a FloatingButtonGroup should change depending on its position.

(Note that in the documentation there's a problem because both buttons have the focus and there should only be one. Link to other issue : https://github.com/twentyhq/twenty/issues/3161)

## Expected behavior

We need to copy the behavior of a FloatingIconButton in a FloatingIconButtonGroup, see `FloatingIconButton.tsx`

image

You can see that the button in the middle has different border-radius because its position is `middle`.

### PR DESCRIPTION
**Pull request by a-kud**
Fixes #2800

before
imagen

after

### CODE REVIEW

**Comment by lucasbordeau** [comment]
@a-kud Thanks for your contribution, the original issue was a bit misleading, I updated it with more context so you can make the relevant changes.

File: packages/twenty-front/src/modules/ui/input/button/components/FloatingButtonGroup.tsx [hunk 1]
```diff
@@ -5,6 +5,14 @@ import { FloatingButtonPosition, FloatingButtonProps } from './FloatingButton';
 
 const StyledFloatingButtonGroupContainer = styled.div`
   backdrop-filter: blur(20px);
+  background: ${({ theme }) => `linear-gradient(
```
Comment (lucasbordeau) [inline packages/twenty-front/src/modules/ui/input/button/components/FloatingButtonGroup.tsx:8]: We don't want a linear-gradient here.

**Review by lucasbordeau — CHANGES_REQUESTED** [review]
We're good with the position.

Here is the reference figma for Floating Buttons, please update the CSS to match the different states accordingly, the default state shouldn't have a blue border : https://www.figma.com/file/xt8O9mFeLl46C5InWwoMrN/Twenty?type=design&node-id=9580-52280&mode=dev

**Comment by charlesBochet** [comment]
@a-kud Any update on this one? :)

**Comment by a-kud** [comment]
I have a high workload right now, feel free to use my PR as a starter to fix the issue.

**Comment by charlesBochet** [comment]
Ok, no worries! We can pick this back a little later :)

**Pull request closed without merging**